test/integration/targets/aws_codepipeline/tasks/main.yml

```diff
@@ -0,0 +1,134 @@
+---
+# tasks file for aws_codepipeline
+
+- name: Run aws_codebuild integration tests.
+
+  block:
+
+    # ==================== preparaions ========================================
+
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{ aws_access_key }}"
+          aws_secret_key: "{{ aws_secret_key }}"
+          security_token: "{{ security_token }}"
+          region: "{{ aws_region }}"
+      no_log: yes
+
+    - name: create IAM role needed for CodePipeline
+      iam_role:
+        name: "{{ resource_prefix }}-codepipeline-service-role"
```

Comment:
The tests currently fail on this task, as the CI running gets a AWS permission error when attempting to create the iam_role. Not sure how this is achieved in other tests.

Reply:
In other tests we add the ability to create an IAM role, but only to attach a specific subset of policies, such as "S3ReadOnly" or similar. Pretty much the minimal permissions necessary to exercise the module. Is there an existing managed policy that grants the perms that CodePipeline needs for these tests?

Reply:
Actually I don't need any permission policy for this role, just the 'assume' policy that I have added already to make it possible for the code pipeline to use the role. But as the test just creates the project but doesn't execute it it needs no permissions to any services.

Reply:
I've added the `needs_ci_update` label to this PR, which puts it on the list of PRs which are awaiting updates to our CI system.

Reply:
Perfect. Thanks!

Reply:
this is typically resolved by adding an `ansible-test-` prefix before the {resource_prefix}. whats odd is i don't see which permissions you were intending to use for codepipeline in terms of vetting other bits, although i guess perhaps it doesn't actually need any since the pipeline isn't executing.

Reply:
Thanks for the hint. Will try that. Yes the pipeline does not execute but still needs to be set up and therefore needs an existing IAM role, hence I have to create one first. As explained above already.